src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/Providers/AzureSqlPsBackupProvider.cs

```diff
@@ -204,9 +204,8 @@ public AzureRmRecoveryServicesBackupSchedulePolicyBase GetDefaultSchedulePolicyO
         public AzureRmRecoveryServicesBackupRetentionPolicyBase GetDefaultRetentionPolicyObject()
         {
             AzureRmRecoveryServicesBackupSimpleRetentionPolicy defaultRetention = new AzureRmRecoveryServicesBackupSimpleRetentionPolicy();
-            defaultRetention.RetentionDuration = new Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.RetentionDuration();
-            defaultRetention.RetentionDuration.RetentionDurationType = Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.RetentionDurationType.Days;
-            defaultRetention.RetentionDuration.RetentionCount = 180;
+            defaultRetention.RetentionDurationType = Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.RetentionDurationType.Months;
```

Comment:
This is just default setting. it can have in months, years, weeks as mentioned in mail. Max allowed is 10 years.